### PR TITLE
feat(tslint): Remove tslint to keep only eslint as tslint is deprecated

### PR DESCRIPTION
**Pull request by benoitf**
### What does this PR do?
Remove usage of tslint (which is deprecated)
Add a new header rule coming from eslint to also automatically fix the header.

### Screenshot/screencast of this PR
N/A

### What issues does this PR fix or reference?
Fix https://github.com/eclipse/che/issues/18308

### How to test this PR?
It's only style configuration, so happy path should still pass.

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [x] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [x] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [x] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [x] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### Happy Path Channel

HAPPY_PATH_CHANNEL=stable

Change-Id: Ie5d97432975cfc9a4711946d5a65436f12c4b8be
Signed-off-by: Florent Benoit <fbenoit@redhat.com>

### CODE REVIEW

File: plugins/workspace-plugin/src/askpass-main.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// tslint:disable:file-header
+/* eslint-disable header/header */
```
Comment (vzhukovs) [inline plugins/workspace-plugin/src/askpass-main.ts:1]: To leave an origin license header it is enough to provide this type of directive? Just curious.

Comment (benoitf) [reply plugins/workspace-plugin/src/askpass-main.ts:1]: yes it is.
Also here I just renamed the `tslint` rule to `eslint` rule.

**Review by vzhukovs — APPROVED** [review]
Looks good 👍

File: plugins/workspace-plugin/src/askpass-main.ts (hunk 1 above)
Comment (vzhukovs) [reply plugins/workspace-plugin/src/askpass-main.ts:1]: Got it.

**Review by RomanNikitenko — APPROVED** [review]

**Review by azatsarynnyy — APPROVED** [review]
thx!

**Pull request merged**